### PR TITLE
Load test report 29-11-23

### PR DESCRIPTION
**Pull request by BertovDev**

### CODE REVIEW

**Review by fkrause98 — APPROVED** [review]

**Comment by manucamejo** [comment]
@BertovDev @fkrause98 did you start 400 games or 40 games with 10 players? I think the filenames are wrong.

**Comment by BertovDev** [comment]
> @BertovDev @fkrause98 did you start 400 games or 40 games with 10 players? I think the filenames are wrong.

You are right, the filename is wrong. I will fix it.

File: load_test/reports/2023_11_29/report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+- Game server is running without dirty scheduler on branch `main #02abd12`
+
+
+# 5 games of 10 players each, 10 minutes.
+
+![Htop during Load Test](img/5_games_10_min.png)
+
+# 10 games, 10 players each, 10 mins.
+
+![Htop during Load Test](img/100_games_10_min.png)
```
Comment (agustinesco) [inline load_test/reports/2023_11_29/report.md:19]:
```suggestion
![Htop during Load Test](img/10_games_10_min.png)
```

File: load_test/reports/2023_11_29/report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Tests were ran using an [AMD ryzen 3600](https://www.amd.com/en/support/cpu/amd-ryzen-processors/amd-ryzen-5-desktop-processors/amd-ryzen-5-3600) with 6 physical cores for the server running the load test and an [Intel XEON Gold 5412U](https://www.intel.la/content/www/xl/es/products/sku/232374/intel-xeon-gold-5412u-processor-45m-cache-2-10-ghz/specifications.html) with 24 physical cores.
```
[the remark this answers is not in the document]
Comment (BertovDev) [reply load_test/reports/2023_11_29/report.md:1]: Are you talking about the commit hashes ?

**Review by agustinesco — APPROVED** [review]
LGTM!

**Review by manucamejo — APPROVED** [review]

**Pull request merged**